### ISSUE TITLE
no CSV-File attached in Reminder-Email

### ISSUE THREAD

**Issue by micha68g** [oliverklee/ext-seminars#216]
I've set:
plugin.tx_seminars {
addRegistrationCsvToOrganizerReminderMail = 1
//and also
fieldsFromFeUserForEmailCsv = name,company,address,zip,city,country,telephone,email
fieldsFromAttendanceForEmailCsv = uid,price,total_price,notes
}
When I trigger the cronjob, I get the mail with the latest registrations but no CSV-file ist attached...
Is this an issue or did I configurate something wrong?

**Comment by oliverklee** [comment]
Hi @micha68g, thanks for reporting this.

Could you please check (using the TS object browser) that this TypoScript is active on the page that you have set in the Scheduler task?

**Comment by micha68g** [comment]
yes. It shows up in the Setup-Tree in the values for [tx_seminars]
(using the TS object browser)

**Comment by oliverklee** [comment]
Thanks for the feedback. Would you be willing to provide a pull request with a fix (and regression test)?

**Comment by oliverklee** [comment]
I've removed the CSV attachment. Closing.
